cmd/contact-auditor/main.go

```diff
@@ -131,7 +131,7 @@ func (c contactAuditor) run(ctx context.Context, resChan chan *result) error {
 		}
 	}
 	// Ensure the query wasn't interrupted before it could complete.
-	err = rows.Close()
+	err = rows.Close() //nolint:sqlclosecheck // this is fine
```

Comment:
nit: I'd prefer a slightly more verbose explanation of why sqlclosecheck thinks this is not fine and why it actually is, rather than just declaring it to be so

Reply:
heh, I was going to write something like "this code will be deleted soon" but I can go into more detail :D